packages/core/src/configuration/Configuration.ts

```diff
@@ -0,0 +1,41 @@
+import { CredentialProvider, Credentials } from '../credentials-provider';
+import { container, DependencyContainer, injectable } from 'tsyringe';
+
+export interface SdkConfiguration {
+  credentials?: Credentials;
+  bus?: string;
+  api?: string;
+}
+
+export interface ConfigurationOptions extends SdkConfiguration {
+  credentialProviders?: Array<CredentialProvider>;
```

Comment:
Get rid of `SdkConfiguration`. It is useless.

```suggestion
  credentials?: Credentials;
  bus?: string;
  api?: string;
  credentialProviders?: Array<CredentialProvider>;
```